sig-docs/annual-report-2021.md

```diff
@@ -0,0 +1,98 @@
+# SIG Docs 2020 Annual Report
+
+**Authors:**
+
+Irvi Aini ([@irvifa](https://github.com/irvifa)),
+Jim Angel ([@jimangel](https://github.com/jimangel)),
+Kaitlyn Barnard ([@kbarnard10](https://github.com/kbarnard10))
+
+- [Operational](#operational)
+- [Membership](#membership)
+- [Current initiatives and project health](#current-initiatives-and-project-health)
+
+## Operational
+
+**How are you doing with operational tasks in [sig-governance.md](/committee-steering/governance/sig-governance.md)?**
+
+  - **Is your README accurate? have a CONTRIBUTING.md file?**
+
+    Yes. We also have `Contributing.md` file.
+  - **All subprojects correctly mapped and listed in [sigs.yaml](/sig-list.md)?**
+
+    Yes.
+      
+  - **What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date?
+    Are you keeping recordings up to date/trends in community members watching recordings?**
+    
+    Quite small (usually 5-10 people) and generally we try to make people feel comfortable to give their opinion.
+    Meeting notes are up to date. We have recording of the meetings uploaded.
+
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired?
+Are OWNERS files up to date in these areas?**
+
+<!-- TODO -->
+
+**How does the group get updates, reports, or feedback from Working Groups? Are there any springing up or being retired? 
+Are OWNERS files up to date in these areas?**
+
+<!-- TODO -->
```

Comment:
@sftim Once we have the thought of doing clean up of approver that's not active anymore on the shift, is this already done? Any specific procedure that we need to share?

Reply:
I think we decided that OWNER files are up to date.

Reply:
Got it thanks 👍